schemars/tests/ui/repr_non_unit_variant.stderr

```diff
@@ -1,5 +1,5 @@
 error: JsonSchema_repr: must be a unit variant
- --> $DIR/repr_non_unit_variant.rs:7:5
+ --> tests/ui/repr_non_unit_variant.rs:7:5
```

Comment:
strange that other PRs are not failing here ; c.f. https://github.com/GREsau/schemars/actions

maybe this needs to be `$DIR` ?